src/actions/account.js

```diff
@@ -264,8 +264,9 @@ export const updateAccountDelegateStats = account =>
   async (dispatch, getState) => {
     const liskAPIClient = getState().peers.liskAPIClient;
     const { address, publicKey } = account;
+    const networkConfig = getState().network;
     const transaction = await getTransactions({
```

Comment:
You need to pass token.active to the getTransactions method

Reply:
I think I shouldn’t pass token.active there, because the two functions are both for delegates (`updateAccountDelegateStats` and `searchDelegate`), so they should never be called with token=BTC

Reply:
You need to read key of the `lisk` token from token constant list and pass it as an active token value instead of getting that from the redux store.

Reply:
@yasharAyari can you be more specific what you mean, because what you're saying is what I think I did after your comment 5 days ago in this commit https://github.com/LiskHQ/lisk-hub/pull/1955/commits/0fd989b6795d7874093dca995a9412b4371d7cc1